src/Symfony/Component/Routing/Loader/PhpFileLoader.php

```diff
@@ -37,7 +38,21 @@ public function load($file, $type = null)
         $path = $this->locator->locate($file);
         $this->setCurrentDir(dirname($path));
 
-        $collection = self::includeFile($path, $this);
+        // the closure forbids access the private scope in the included file
```

Comment:
`forbids access the private scope` -> `forbids accessing the private scope` or `forbids access to the private scope` ?